http_outlet_test.go

```diff
@@ -89,23 +92,98 @@ func TestOutletEOFRetryMax(t *testing.T) {
 
 	outlet.retryPost(batch)
 	if th.called != config.MaxAttempts {
-		t.Errorf("th.called != %q, == %q\n", config.MaxAttempts, th.called)
+		t.Errorf("th.called != %q, == %d\n", config.MaxAttempts, th.called)
 	}
 
 	if lost := s.Lost.Read(); lost != 1 {
-		t.Errorf("lost != 1, == %q\n", lost)
+		t.Errorf("lost != 1, == %d\n", lost)
 	}
 
 	mrLost := metrics.GetOrRegisterCounter("msg.lost", s.MetricsRegistry)
 	if lost := mrLost.Count(); lost != 1 {
-		t.Errorf("lost != 1, == %q\n", lost)
+		t.Errorf("lost != 1, == %d\n", lost)
 	}
 
 	if msg := logCapture.Bytes(); !bytes.Contains(msg, []byte("EOF")) {
 		t.Errorf("expected log message to contain `EOF`, got %q", msg)
 	}
 }
 
+func TestOutletOtherRetry(t *testing.T) {
+	logLineText := "Hello"
+
+	var reqs int
+	ts := httptest.NewTLSServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		reqs++
+		if reqs >= 3 {
+			return
+		}
+
+		h := w.(http.Hijacker)
+
+		c, _, err := h.Hijack()
+		if err != nil {
+			panic(err)
+		}
+
+		// Produces a "malformed HTTP response" error.
+		c.Write([]byte("bogus"))
+		c.Close()
+	}))
+	defer ts.Close()
+
+	config := newTestConfig()
+	config.LogsURL = ts.URL
+	config.SkipVerify = true
+
+	logCapture := new(bytes.Buffer)
+	s := NewShuttle(config)
+	s.ErrLogger = log.New(io.MultiWriter(logCapture, os.Stdout), "", 0)
```

Comment:
I'd much rather t.Info the buffer.Bytes()

Reply:
Went ahead and dropped the extra output altogether, can address some other time.